modules/ROOT/pages/mule-app-strategies.adoc

```diff
@@ -19,4 +19,4 @@ These topics cover Mule app configurations that you can use with various Mule co
 * xref:target-variables.adoc[Enrich Data with Target Parameters]
```

Comment:
"General Configurations
"These topics cover Mule app configurations that you can use...."

Is there a more specific word than "General"? Are they common, or public, or basic, or....
Also, try to avoid self-reference such as "These topics cover...." How about something like (and I am winging it here) "You can use Mule runtine engine components to configure a variety of Mule applications, such as <list of the top-level apps that follow, or just some of them."

Reply:
I am not working on this topic now. I will capture your feedback for later on.